### DISCUSSION TITLE
external converters after reboot of the host

### DISCUSSION THREAD

**Discussion by borsa93** [Koenkk/zigbee2mqtt#11099] [Devices]
Hi, I'm using the Z2M addon (HA installed with docker on RPi 4) with a custom device (I added the .js file in the configuration.yaml) and everything is working very well until I reboot the host (for some reason). After a reboot I have to put again the .js reference in the config file in order to allow Z2M to recognize it. (it happens only if I reboot the host, not if i reboot only HA)
Is it something normal o is it an issue? Any idea on how to fix it?

Thanks in advance for the answers

**Comment by rst-ack** [comment]
I have the same problem -- Running Hass.IO on a RPi4 using the Z2M add-on. Whenever the Z2M add-on is restarted (either via rebooting the whole host or just the add-on) my external converters would drop off and I need to go back in to the Z2M interface, and add the external converter again.
